src/Surfnet/StepupGateway/U2fVerificationBundle/Tests/Value/KeyHandleTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/**
+ * Copyright 2015 SURFnet bv
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\StepupGateway\U2fVerificationBundle\Tests\Value;
+
+use PHPUnit_Framework_TestCase as TestCase;
+use Surfnet\StepupGateway\U2fVerificationBundle\Value\KeyHandle;
+
+final class KeyHandleTest extends TestCase
+{
+    use ValueObjectTest;
```

Comment:
Might be worth to introduce this in the Middleware as well as we've duplicated these providers there all over the place.